nima/webclient/api/src/main/java/io/helidon/nima/webclient/api/ClientRequest.java

```diff
@@ -375,6 +375,15 @@ default <T> ClientResponseTyped<T> outputStream(OutputStreamHandler outputStream
      */
     T readTimeout(Duration readTimeout);
 
+    /**
+     * Read 100-Continue timeout for this request.
+     * This read timeout is used when 100-Continue is sent by the client, before it sends an entity.
+     *
+     * @param readContinueTimeout read 100-Continue timeout duration
+     * @return updated client request
+     */
+    T readContinueTimeout(Duration readContinueTimeout);
```

Comment:
Maybe reference the configuration option that provides a default value information

Reply:
done